.travis.yml

```diff
@@ -1,20 +1,11 @@
 language: elixir
 
 elixir:
+  - 1.7
   - 1.6
   - 1.5
   - 1.4
 
-otp_release:
```

Comment:
Can you link us to the documentation that explains why we don't need this? If you look around pretty much every Elixir project has these OTP releases defined, including Elixir itself.

Reply:
https://docs.travis-ci.com/user/languages/elixir#specifying-otp-release-version I think I'm reading this correctly...

Reply:
I believe so. As I understand it, if we leave this off, it will pick _one_ OTP version. We are intentionally testing against multiples because it is common in real-world situations to have Elixir and OTP on different versions.